language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -124,11 +186,11 @@ makeLenses ''CekEnv
 
 -- | The monad the CEK machine runs in. State is inside the ExceptT, so we can
 -- get it back in case of error.
-type CekM uni = ReaderT (CekEnv uni) (ExceptT (CekEvaluationException uni) (QuoteT (State ExBudgetState)))
+type CekM uni = ReaderT (CekEnv uni) (ExceptT (CekEvaluationException uni) (State ExBudgetState))
```

Comment:
That's the expected consequence of using the alternative CEK machine, 'cause we don't need to generate fresh variables during evaluation, since the problem of evaluation of polymorphic built-in functions can be solved in a much more straightforward way with the alternative CEK machine. Since values in the alternative CEK machine carry their own environments, we basically don't need to do anything special regarding polymorphic built-in functions. They just work out of the box.